src/main/java/chess/domain/state/Progress.java

```diff
@@ -0,0 +1,42 @@
+package chess.domain.state;
+
+import chess.domain.board.ChessBoard;
+import chess.domain.position.Position;
+import chess.domain.position.Positions;
+import java.util.List;
+
+public class Progress implements GameState {
+    private static final String START_COMMAND = "start";
+    private static final String MOVE_COMMAND = "move";
+    private static final String END_COMMAND = "end";
+
+    private final ChessBoard chessBoard;
+
+    public Progress(ChessBoard chessBoard) {
+        this.chessBoard = chessBoard;
+    }
+
+    @Override
+    public GameState play(List<String> inputCommand) {
+        String command = inputCommand.get(0);
+        if (command.equals(START_COMMAND)) {
```

Comment:
```suggestion
        if (START_COMMAND.equals(command)) {
```
equals비교시 상수가 앞으로 오게하여 혹시 모를 NPE를 예방해주세요!